### DISCUSSION TITLE
Windows 11 Taskbar Styler option to remove the start button from other screens

### DISCUSSION THREAD

**Discussion by f3rn4nd0** [ramensoftware/windhawk-mods#929] [Mod ideas]
Hi Guys, I'm using Windows 11 Taskbar Styler and I saw that it has the option to remove the start bar, but this is for all bars, I use 3 monitors, it would be interesting if it were possible to leave the Start only on the main screen, I do not know if it is possible, but I leave here the idea.

**Comment by m417z** [comment]
Currently there's no good way to do that. I compared the attributes of the main taskbar and the secondary taskbar, and the only difference is the taskbar width in case your monitors have different resolutions. So that can be a workaround. Use UWPSpy to find an attribute that can be used (perhaps `Width` in `Taskbar.TaskbarFrame`) and set it as part of the target. See the mod description to see how attributes can be used in targets.
